generators/connection/index.js

```diff
@@ -213,6 +229,91 @@ module.exports = class ConnectionGenerator extends Generator {
           return true;
         }
       },
+      {
+        type: 'list',
+        name: 'connectionStyle',
+        message: 'How would you like to enter your database credentials?',
+        default: 'connectionString',
+        choices: [
+          { name: 'Connection string', value: 'connectionString' },
+          { name: 'Connection object', value: 'connectionObject' },
+        ],
+        when (current) {
+          const answers = getProps(current);
+          const { adapter, database } = answers;
+
+          return adapter === 'sequelize' && database !== 'mssql';
```

Comment:
I'm currently leaving mssql out because I see that it has special handling where the connection string is broken out into components. Let me know if you think it makes sense to refactor that, too,

Reply:
e.g. force connection object on mssql and fold the `sequelize-mssql.js` template into the regular `sequelize.js` template.